dbochen/KitchenProject
# Create simple fronted in react

Closes #1 

### What was done

Set up a basic React app in Typescript displaying hello world.

### How to test

1. Install `Node.js` -> https://www.liquidweb.com/kb/install-react-js-windows/
1. Cd into `src/main/ui`
1. Run `yarn install && yarn start`
1. Your browser should open automatically on `http://localhost:3000/`
